shell.nix

```diff
@@ -0,0 +1,18 @@
+# TODO: Pin this to a specific version of nixpkgs
```

Comment:
Needs a lot of work but this is a decent start :)

Reply:
One small step for some. One giant leap for us.

Reply:
hahaha so true :laughing: